carts/src/routes/index.ts

```diff
@@ -0,0 +1,61 @@
+import express from "express";
+const routes = express.Router();
+const cartItems = (
+  id: number,
+  product: string,
+  price: number,
+  quantity: number
+) => ({
+  id,
+  product,
+  price,
+  quantity,
+});
+const albums = [
+  cartItems(1, "Love Angel Music Baby", 12.99, 5),
+  cartItems(2, "Paradise", 9.99, 3),
+  cartItems(3, "Tako Tsubo", 8.99, 7),
+];
+routes.get("/", (req, res) => {
+  res.json("hello world");
+});
+routes.get("/cart-items", (req, res) => {
+  res.status(200).json(albums);
+});
+routes.get("/cart-items/:id", (req, res) => {
+  const itemId = +req.params.id;
+  const item = albums.find((item) => item.id === itemId);
+  if (item) {
```

Comment:
Wrap these in try/catch blocks. Youre if/else is mostly serving the same purpose here, but try/catch is the convention.